### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by r-c-k**
Changed the function call from "returns_i64()" to "returns_unit()"

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Good catch, thanks!
@bors r+ rollup

File: library/std/src/primitive_docs.rs [hunk 1]
```diff
@@ -378,7 +378,7 @@ mod prim_char {}
 ///     returns_i64()
 /// };
 /// let is_unit = {
-///     returns_i64();
+///     returns_unit();
```
Comment (JohnTitor) [inline library/std/src/primitive_docs.rs:381]: Oh, it should be better:

```suggestion
///     returns_unit()
```

Comment (JohnTitor) [reply library/std/src/primitive_docs.rs:381]: Or, I'm wondering it's intentional to show that adding `;` to the end of `returns_i64` results `()`.

Comment (r-c-k) [reply library/std/src/primitive_docs.rs:381]: Ah yeah, it probably is 😄

**Pull request closed without merging**